admin/src/Alert.js

```diff
@@ -8,6 +7,7 @@ import {
   faThumbsUp,
   faSpinner,
   faExclamationTriangle } from '@fortawesome/free-solid-svg-icons'
+import React from 'react'
```

Comment:
Is it important that this can't be listed as the first `import`?

Reply:
moved 'em up